projects/packages/my-jetpack/_inc/components/add-license-screen/index.jsx

```diff
@@ -19,14 +18,16 @@ import GoBackLink from '../go-back-link';
  * @returns {object} The AddLicenseScreen component.
  */
 export default function AddLicenseScreen() {
-	useEffect( () => {
-		const { apiRoot, apiNonce } = window?.myJetpackRest || {};
-		restApi.setApiRoot( apiRoot );
-		restApi.setApiNonce( apiNonce );
-	}, [] );
-
 	const { recordEvent } = useAnalytics();
-	const { availableLicenses, fetchingAvailableLicenses } = useAvailableLicenses();
+	const { data: availableLicenses = [], isLoading } = useJetpackApiQuery(
+		'available licenses',
+		async api => {
+			const result = await api.getUserLicenses();
+			return result?.items?.filter(
+				( { attached_at, revoked_at } ) => attached_at === null && revoked_at === null
+			);
+		}
+	);
```

Comment:
This might be enough to move it to it's own file 🤔  It's only used here so I don't think its necessarily required right now but wanted to just mention it

Reply:
Hmm, in a way it's just using established external api to get data and filtering through it.

I think we could maybe move `result?.items?.filter(
				( { attached_at, revoked_at } ) => attached_at === null && revoked_at === null
			);` out of that inline function and keep it `api => api.getUserLicenses()` if you think it's more readable

Reply:
No I like the readability of this 🤔 

It's arbitrary, but it just seems too "long" for me to have in the consumer 😅 . That is just my personal preferences showing though so if it's not worth moving to a file in your opinion I am okay keeping it here